mne_bids/write.py

```diff
@@ -608,12 +608,15 @@ def _write_raw_brainvision(raw, bids_fname):
         raise ImportError('pybv >=0.2.0 is required for converting ' +
                           'file to Brainvision format')
     from pybv import write_brainvision
-    events, event_id = events_from_annotations(raw)
+    events, _ = events_from_annotations(raw)
+    events[:, 0] -= 2334  # raw.first_samp
```

Comment:
why is this hardcoded?

Reply:
To show an example that it's messed up :)

Reply:
Whatever number you put in there, it doesn't change the events